haute-ecole-de-gestion-de-geneve-iso-690.csl

```diff
@@ -356,15 +415,14 @@
               <text macro="title" suffix=". "/>
             </if>
           </choose>
-          <text variable="version" prefix="version " suffix=". "/>
+          <text variable="version" prefix="Version " suffix=". "/>
```

Comment:
`version` now is a term you can use. Also, do you still need this now that `software` is available as an item type & mapped from Zotero?

Reply:
With `software` available as a doctype in CSL, this line is not useful anymore. I will remove it.